homeassistant/components/climacell/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "ClimaCell",
+  "config": {
+    "step": {
+      "user": {
+        "description": "If [%key:component::climacell::config::step::user::data::latitude%] and [%key:component::climacell::config::step::user::data::longitude%] are not provided, the default values in the Home Assistant configuration will be used. An entity will be created for each forecast type but only the ones you select will be enabled by default.",
+        "data": {
+          "name": "Name",
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "latitude": "Latitude",
+          "longitude": "Longitude"
```

Comment:
I am translating this integration in Lokalise.
Why is `Name`, `Latitude` and `Longitude` not `[%key:common::config_flow::data::XXX%]`.
Translating these new strings takes longer than copying the reference.

Reply:
That's an oversight. PR is welcome to fix it. :+1:

Reply:
submitting shortly, thanks for catching that!

Reply:
I often see this in other integrations as well. Can I open one PR to solve this in multiple integrations or does each integration have to be in a separate PR?

Reply:
Please do one integration per PR. :+1: